src/rt.erl

```diff
@@ -279,17 +279,20 @@ rpc_get_env(Node, [{App,Var}|Others]) ->
 
 -spec connection_info(node() | [node()]) -> interfaces() | conn_info().
 connection_info(Node) when is_atom(Node) ->
-    {ok, [PB_Info]} = get_pb_conn_info(Node),
-    {ok, [HTTP_Info]} = get_http_conn_info(Node),
-    Info0 = [{pb, PB_Info}, {http, HTTP_Info}],
-    Info1 = case get_https_conn_info(Node) of
-        undefined -> Info0;
-        {ok, [{HTTPS_IP, HTTPS_Port}]} -> [{https, {HTTPS_IP, HTTPS_Port}} | Info0]
+    {ok, [HTTP_Info0]} = get_http_conn_info(Node),
+    HTTP_Info = [{http, HTTP_Info0}],
+    {ok, [PB_Info0]} = get_pb_conn_info(Node),
+    PB_Info = [{pb, PB_Info0}],
```

Comment:
Not to be totally anal, but maybe list PB_Info after HTTPS_Info, since, you know, we are wanting it to be ordered (I realize we achieve order by the ++ below)